hbase-server/src/test/java/org/apache/hadoop/hbase/security/TestUsersOperationsWithSecureHadoop.java

```diff
@@ -65,11 +67,19 @@
   private static String CLIENT_NAME;
 
   @BeforeClass
-  public static void destroyAndSetup() throws Exception {
-    //destroy localhost kerberos users
-    Process process = Runtime.getRuntime().exec(new String[]{"bash", "-c", "kdestroy"});
-    process.waitFor(2, TimeUnit.SECONDS);
```

Comment:
This previous `waitFor` call doesn't check the return value. It's possible the call failed but the test didn't notice.

Reply:
So, the previous exec w/ its kdestroy is better than klist with a 'notice' IMO. What tends to happen is you run test suite and then one hour in, it will fail with complaint about unmatched user for old ticket. WIth kdestroy, that doesn't happen. Means old ticket gets killed which should be fine most of the time -- you just re-kinit  (unless you running test suite on some critical system).

Nick's idea of narrowing test so it only concerned about test ticket would be best. Not sure how to do that though myself.